README.md

```diff
@@ -1,26 +1,255 @@
+
+In addition, we have created a pre-built image with both Raspbian Stretch and ROS already installed. You can download this image as a last resort if you can’t get Raspbian and ROS installed and working on your own, but please note that you will miss out on some the learning opportunities that come with debugging a software installation by doing so. 
+
+- [Tutorial](https://projects.raspberrypi.org/en/projects/raspberry-pi-getting-started) for installing the latest version of Raspbian on a new Raspberry Pi
+- Direct downloadable Raspbian Stretch [image](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip)
+- Custom JPL [image](https://drive.google.com/drive/folders/1RbYWDRthpcktqkPEHb7qVqDiy27EiAc1) with both Raspbian Stretch and ROS pre-built
+
+**NOTE: If you install the Custom JPL image with Raspbian Stretch and ROS pre-built, skip to step 2.4.**
+
+Once you finish the above tutorial and install Raspbian Stretch, you should be able to boot your Raspberry Pi and see a desktop!
+
+
+
+
+
+### 2.2 Installing ROS
+
+
+#### 2.2.1 ROS Melodic on Ubuntu 18.04
+
+Use these instructions if you followed subsection 2.1.1 above.
+
+The below steps are based off of [these instructions](http://wiki.ros.org/melodic/Installation/Ubuntu). Consult them for more details.
+
+Setup
+```
+# Setup your computer to accept software from packages.ros.org
+sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+
+# set up keys
+sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+```
+
+Installation
+```
+# make sure your Debian package index is up-to-date
+sudo apt update
+
+# desktop-Full Install: (Recommended) : ROS, rqt, rviz, robot-generic libraries, 2D/3D simulators and 2D/3D perception
+sudo apt install ros-melodic-desktop-full
+```
+
+Dependencies for building packages
+```
+# install 'em
+sudo apt install python-rosdep python-rosinstall python-rosinstall-generator python-wstool build-essential
+
+# initialize rosdep
+sudo apt install python-rosdep
+sudo rosdep init
+rosdep update
+
+# sensor msgs dependencies
+rosinstall_generator sensor_msgs --rosdistro melodic --deps --wet-only --tar >melodic-sensor_msgs-wet.rosinstall
+wstool init src melodic-sensor_msgs-wet.rosinstall
+```
+
+#### 2.2.2 ROS Kinetic on Raspbian
+
+Use these instructions if you followed subsection 2.1.2 above.
+
+If you do not download the pre-built Raspbian Stretch / ROS image, you will need to build ROS yourself.
+
+First, make sure you update all the packages on your pi:
+
+```
+sudo apt-get update
+sudo apt-get upgrade
+```
+
+(The above commands may take a few minutes to run.)
+
+Next, we will install ROS (and specifically, the version called ’Kinetic’).
```

Comment:
```suggestion
Next, we will install ROS (and specifically, the version called ’Melodic’).
```
Ubuntu18.04 --> ROS Melodic [rep0003: target platforms](https://www.ros.org/reps/rep-0003.html)

Reply:
Nah, this should be Kinetic here. This is for ROS Kinetic on Raspbian, per the original instructions. The more relevant consideration is if we want to still support Kinetic on Raspian...which I say we shouldn't, if we're transitioning to only supporting 18.04. I had a bunch of trouble trying to install Kinetic on 18.04.

So...see the discussion below.